src/util/taskManager.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+
+import {
+  type TaskId,
+  type TaskReporter,
+  ScopedTaskReporter,
+  SilentTaskReporter,
+} from "./taskReporter";
+
+export type TaskNode = {|
+  reporter: TaskReporter,
+  children: Map<TaskId, TaskNode>,
+|};
```

Comment:
Why is this type exported?

If it's just because the private methods of TaskManager export it, maybe we can name it `_TaskNode` or such to communicate that it's an implementation detail?